content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -94,7 +92,17 @@ run the `kfctl apply` command:
 ```bash
 mkdir -p ${KF_DIR}
 cd ${KF_DIR}
-kfctl apply -V -f ${CONFIG_URI}
+
+# Recommended: Download the config file and change the default login credentials.
+wget -O kfctl_existing_arrikto.yaml $CONFIG_URI
+export CONFIG_FILE=${KF_DIR}/kfctl_existing_arrikto.yaml
```

Comment:
One more thing we need to change: The name of the local config file created by kfctl now also has a version number attached. So, in this case, the local file name is `kfctl_existing_arrikto.0.7.0.yaml`.  While you're busy in this page, would you change it throughout?

I need to update the rest of the docs. I'll propose with a shortcode (that is, a variable like we use for the config URI) to make this easier in future iterations. For now, though, it's good to just change all occurrences on the page.

Reply:
Actually, this command:
`wget -O kfctl_existing_arrikto.yaml $CONFIG_URI`

ensures that the downloaded filename is kfctl_existing_arrikto.yaml.
So there is no need to change it everywhere.
Is this ok?

Reply:
I saw that, but there are 2 potential issues. First, we need to remove the statement that this is recommended, so that everyone does it. Second, I think kfctl may go ahead and create the versioned file anyway?

Reply:
Ok on the recommended part.
For the kfctl part, it should keep the filename the same.

Reply:
Update: I removed the recommended wording.
cc @sarahmaddox

Reply:
OK, sounds good.